tests/snappi_tests/bgp/test_bgp_outbound_uplink_po_member_flap.py

```diff
@@ -23,32 +23,22 @@
 ITERATION = 1
 ROUTE_RANGES = [{
                     'IPv4': [
-                        ['100.1.1.1', 24, 500],
-                        ['200.1.1.1', 24, 500]
+                        ['100.1.1.1', 24, 15000],
+                        ['200.1.1.1', 24, 15000]
                     ],
                     'IPv6': [
-                        ['5000::1', 64, 500],
-                        ['4000::1', 64, 500]
+                        ['5000::1', 64, 15000],
+                        ['4000::1', 64, 15000]
                     ],
-                },
-                {
-                    'IPv4': [
-                        ['100.1.1.1', 24, 2500],
-                        ['200.1.1.1', 24, 2500]
-                    ],
-                    'IPv6': [
-                        ['5000::1', 64, 2500],
-                        ['4000::1', 64, 2500]
-                    ],
-            }]
+                }]
 
 
 def test_bgp_outbound_uplink_po_member_flap(snappi_api,                                     # noqa: F811
                                             multidut_snappi_ports_for_bgp,                # noqa: F811
                                             conn_graph_facts,                             # noqa: F811
                                             fanout_graph_facts_multidut,                   # noqa: F811
                                             duthosts,
-                                            creds):
+                                            creds, record_property):
```

Comment:
can we bring this record_property change to all tests?

Reply:
will remove this one